### PR TITLE
Update wallets.md

### PR DESCRIPTION
**Pull request by Dorreguinho**

### CODE REVIEW

**Review by filippoweb3 — COMMENTED** [review]
thanks for proposing this addition. Although I can see the project received W3F funding, the app does not seem to be a wallet specific to the Polkadot/Kusama ecosystem, and IMO, the addition of this project to this page is not justified. I will pass this to @DrW3RK for final decision. Thanks for your understanding.

**Comment by Dorreguinho** [comment]
Hi @filippoweb3, thanks for your comment! It's not specific because we have a multichain approach, but we're really interested in getting more and more involved with the Polkadot Ecosystem, for that reason the application.

Hope that we can overcome this. Btw, we're super interested in discussing some kind of collaboration or action to get more involved in the ecosystem.

Kind regards, Agustin.

**Comment by Dorreguinho** [comment]
Hello @filippoweb3 & @DrW3RK, have you any update on this?

**Comment by DrW3RK** [comment]
Hi @Dorreguinho, We are currently reviewing your wallet implementation. Please give us some additional time for this. 

We are excited that you want to be more involved in the Polkadot ecosystem. Could you please send an email to me at radha@web3.foundation and I will put you in touch with the relevant team!

File: docs/general/wallets.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ wallets through Bluetooth connectivity.
 | [Nova Wallet](https://novawallet.io/)\*       | iOS, Android                                                    | **Yes**, **Yes**             | **Yes** | **Yes**    | **Yes**        | **Yes**    | [Nova Wallet features](#nova-wallet)         |
 | [Polkawallet](https://polkawallet.io/)        | iOS, Android                                                    | **Yes**, **Yes**             | No      | **Yes**    | No             | **Yes**    | [Polkawallet features](#polkawallet)         |
 | [SubWallet](https://subwallet.app/)           | [iOS (Testflight)](apple.co/3WafLpm), [Android](bit.ly/3DE2Dlg) | **Yes**, **Yes**             | **Yes** | **Yes**    | **Yes**        | No         | [SubWallet features](#subwallet)             |
+| [Citadel.one](https://app.citadel.one/)           | [iOS (Testflight)](apple.co/3PWc6s9), [Android](https://bit.ly/3N9ZtrH) | **Yes**, **Yes**             | **Yes** | **No**    | **Yes**        | Yes         | [Citadel.one Features](#citadel-one)             |
```
Comment (profradha) [inline docs/general/wallets.md:76]:
```suggestion
| [Citadel.one](https://app.citadel.one/)           | [iOS (Testflight)](apple.co/3PWc6s9), [Android](https://bit.ly/3N9ZtrH) | **Yes**, No            |  No  |  No    | **Yes**       | No         | [Citadel.one Features](#citadel-one)             |
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
I tested the wallet on iPhone and I see that the wallet allows for balance transfers and staking. Although, I do not see an option to choose a nomination pool. I see there are apps for participation on governance on other chains, but I do not see an app for Polkadot. Also, where can I see/interact with NFTs on my Polkadot wallet? 

Please let me know if I'm mistaken with my testing. Looking forward to your response! Also, is the wallet software open source?

**Comment by Dorreguinho** [comment]
@DrW3RK Hello! You're right, I've a typo when I was completing the table. Thanks for your review!

Btw, I just sent you an email so you can put me in touch :)

Thanks again for your kindness!

**Comment by Dorreguinho** [comment]
@DrW3RK Hey hope that you're doing well! 
I'd like to know some updates on this, also I sent you an email some weeks ago to get in contact with the foundation.

Thanks a lot! Have an awesome week :)

**Comment by DrW3RK** [comment]
No plans to modify the wallets page to include closed source projects at this time.

**Pull request closed without merging**